tests/benchmark/server.py

```diff
@@ -0,0 +1,22 @@
+import asyncio
+
+from aiohttp import web
```

Comment:
Could you use another library here? e.g. FastAPI, to avoid any possible internal optimization between aiohttp's client and server.

Reply:
Hmm it shouldn't make any difference as its running in a separate process as a normal HTTP server. I wouldn't want to include FastAPI into the benchmark as it is a pretty thick framework.

Reply:
Switched to using `uvicorn` (which FastAPI also uses in background). It makes no difference when rerunning benchmarks.